app/models/content_item.rb

```diff
@@ -10,6 +10,8 @@ def initialize(content_store_response)
     @title = content_store_response["title"]
     @base_path = content_store_response["base_path"]
     @locale = content_store_response["locale"]
+    @last_updated = content_store_response["public_updated_at"]
```

Comment:
Hmm. I sort of think here we should be keeping the attributes named the same as they are in the content store - like, this should be `public_updated_at`